.github/workflows/main.yaml

```diff
@@ -1,4 +1,4 @@
-name: CI
+name: Automation
```

Comment:
nit: maybe rename to "Oryx-SDK-Automation" for clarity?

Reply:
Yup, I've updated the name